blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -267,6 +269,19 @@ public void testHttpAggregatedMultiIngestionHappyCase() throws Exception {
                 locator1, new Range(1439231323000L, 1439231325000L), CassandraModel.getColumnFamily(BluefloodCounterRollup.class, Granularity.FULL));
         Assert.assertEquals(1, points1.getPoints().size());
 
+        final Locator locator2 = Locator.createLocatorFromPathComponents("5405577", "call_xyz_api");
+        Points<BluefloodEnumRollup> points2 = new FakeEnumAstyanaxReader().getInstance().getDataToRoll(BluefloodEnumRollup.class,
+                locator2, new Range(1439231323000L, 1439231325000L), CassandraModel.getColumnFamily(BluefloodEnumRollup.class, Granularity.FULL));
+        Assert.assertEquals(1, points1.getPoints().size());
+        for (Long timestamp : points1.getPoints().keySet()) {
+            BluefloodEnumRollup er = points2.getPoints().get(timestamp).getData();
+            for (String value : er.getStringEnumValuesWithCounts().keySet()) {
+                Assert.assertEquals("500",value);
```

Comment:
it's in the sample payload file (blueflood-http/src/test/resources/sample_multi_aggregated_payload.json). It's an integration test, ingesting the sample payload first and then querying for it, which is why it's in the metrics_enum CF.